mail_thread_create_nolog/models/mail_message.py

```diff
@@ -64,5 +78,5 @@ def _messages_for_record(self, domain):
             if model and res_id:
                 break
         if model and res_id:
-            return self.env[model].browse(res_id).exists()
+            return self.env[model].sudo().browse(res_id).exists()
```

Comment:
why using sudo here?

Reply:
I have a error before. But I check again, it works. So I changed it back